pages/models/regulations/AC1C/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Standard voltage regulator model
+tags: ["Voltage regulator", "AC1C", "generic"]
+---
+# AC1C model
+
+## Context
+
+This voltage regulator model has been developed by RTE.
+
```

Comment:
can you add material in the ## Model use, assumptions, validity domain and limitations section?

Reply:
I have left this part empty.